aether-kernel/aether/kernel/api/models.py

```diff
@@ -99,14 +99,12 @@ class Meta:
         ]
 
 
-class Mapping(ExportModelOperationsMixin('kernel_mapping'), TimeStampedModel):
+class MappingSet(ExportModelOperationsMixin('kernel_mappingset'), TimeStampedModel):
 
     id = models.UUIDField(primary_key=True, default=uuid.uuid4)
     revision = models.TextField(default='1')
     name = models.CharField(max_length=50, null=False, unique=True)
-
-    definition = JSONField(blank=False, null=False)
-
+    input = JSONField(blank=False, null=False)
```

Comment:
Is this really necessary?

Reply:
Being that contracts (schemas + mappings) are now logically grouped by the input, i think it should be stored if available. It also helps to recreate a complete pipeline (input + contracts) on the ui for better UX when adding a new contract to a set.

Reply:
In that case it shouldn't be mandatory. The ODK module doesn't have an input, but it creates the artefacts...